packages/devextreme/js/__internal/ui/chat/messagegroup.ts

```diff
@@ -136,6 +148,10 @@ class MessageGroup extends Widget<Properties> {
     $information.appendTo(this.element());
   }
 
+  _updateIsLastOptionOfLastMessage(isLast: boolean): void {
```

Comment:
It's a "public" method
```suggestion
  updateIsLastOptionOfLastMessage(isLast: boolean): void {
```

Reply:
0e11dd012baad48fce62eeab905c60abc39aa9d5